modules/deployment-plug-in-cluster.adoc

```diff
@@ -10,29 +10,86 @@ After pushing an image with your changes to a registry, you can deploy the plugi
 
```

Comment:
Should we also change the name of the file to `deployment-plugin-cluster.adoc`

Reply:
updated and also updated the others as well. Thanks!